modules/local/find_primer_cutoffs/main.nf

```diff
@@ -4,8 +4,8 @@ process FIND_PRIMER_CUTOFFS {
     tag "$meta.id"
     label 'very_light'
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        "https://depot.galaxyproject.org/singularity/mi-pimento:1.0.0--pyhdfd78af_0":
-        "biocontainers/mi-pimento:1.0.0--pyhdfd78af_0"}"
+        "https://depot.galaxyproject.org/singularity/mi-pimento:${params.pimento_version}":
+        "biocontainers/mi-pimento:${params.pimento_version}"}"
```

Comment:
Just a though. I feel the version of the tools shouln't be an argument, I understand that having this makes our life much easier.

A better approach would be to automate version bumps across our repos... a big piece of work. Ideally every time we push a new version of pimento or the toolkit we bump the version in nf-modules, which then bumps the version of the pipelines.